.flowconfig

```diff
@@ -26,6 +26,7 @@ shells/dev/build/*
 
 [options]
 esproposal.class_instance_fields=enable
+esproposal.optional_chaining=enable
```

Comment:
I wonder why is this still here, I remember reverting because it broke compilation of tests.